### PR TITLE
Label fix

### PR DESCRIPTION
**Pull request by sjwoodman**
Improved logging and moved to Log4j

Fixed bugs where not all files were looking for the correct labels

Simplified by using container name rather than an extra label

### CODE REVIEW

File: Operator/src/main/java/io/streamzi/ev/Manager.java [hunk 1]
```diff
@@ -3,22 +3,21 @@
...
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
 /**
  * Start the ConfigMap Watcher
  */
 public class Manager {
 
-    private static final Logger logger = Logger.getLogger(Manager.class.getName());
+    private final static org.apache.logging.log4j.Logger logger = LogManager.getLogger(Manager.class);
```
Comment (matzew) [inline Operator/src/main/java/io/streamzi/ev/Manager.java]: why the full package?

Comment (sjwoodman) [reply Operator/src/main/java/io/streamzi/ev/Manager.java]: Didn't spot that - IDE artefact when changing to log4j. Will fix.

File: Operator/pom.xml [hunk 2]
```diff
@@ -14,12 +14,36 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>${log4j.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```
Comment (matzew) [inline Operator/pom.xml:36]: what's wrong w/ JUL ? :-)

File: Operator/src/main/java/io/streamzi/ev/operator/DeploymentConfigOperator.java [hunk 3]
```diff
@@ -6,22 +6,24 @@
...
-import java.util.logging.Logger;
 
 /**
  * EnvironmentVariableOperator that will take the payload of a DeploymentConfig and check to see if there are ConfigMaps
  * containing environment variables that should be applied to it
  */
 public class DeploymentConfigOperator implements EnvironmentVariableOperator<DeploymentConfig> {
 
-    private final Logger logger = Logger.getLogger(DeploymentConfigOperator.class.getName());
+    private final static org.apache.logging.log4j.Logger logger = LogManager.getLogger(DeploymentConfigOperator.class);
```
Comment (matzew) [inline Operator/src/main/java/io/streamzi/ev/operator/DeploymentConfigOperator.java]: full package

File: Operator/src/main/java/io/streamzi/ev/operator/DeploymentOperator.java [hunk 4]
```diff
@@ -6,22 +6,24 @@
...
-import java.util.logging.Logger;
 
 /**
  * EnvironmentVariableOperator that will take the payload of a DeploymentConfig and check to see if there are ConfigMaps
  * containing environment variables that should be applied to it
  */
 public class DeploymentOperator implements EnvironmentVariableOperator<Deployment> {
 
-    private final Logger logger = Logger.getLogger(DeploymentOperator.class.getName());
+    private final static org.apache.logging.log4j.Logger logger = LogManager.getLogger(DeploymentOperator.class);
```
Comment (matzew) [inline Operator/src/main/java/io/streamzi/ev/operator/DeploymentOperator.java]: full package

File: Operator/src/main/java/io/streamzi/ev/watcher/ConfigMapWatcher.java [hunk 5]
```diff
@@ -10,20 +10,18 @@
...
 import java.util.Map;
-import java.util.logging.Logger;
 
 /**
  * Watch for changes in a ConfigMap and push them to an EnvironmentVariableOperator for applying changes to the Environment Variables of containers
  */
 public class ConfigMapWatcher implements Watcher<ConfigMap>, Runnable {
 
-    private final static Logger logger = Logger.getLogger(ConfigMapWatcher.class.getName());
+    private final static org.apache.logging.log4j.Logger logger = LogManager.getLogger(ConfigMapWatcher.class);
```
Comment (matzew) [inline Operator/src/main/java/io/streamzi/ev/watcher/ConfigMapWatcher.java]: full package

File: Operator/src/main/java/io/streamzi/ev/watcher/DeploymentConfigWatcher.java [hunk 6]
```diff
@@ -11,16 +11,16 @@
...
 import java.util.Map;
-import java.util.logging.Logger;
 
 /**
  * Watch for changes in a ConfigMap and push them to an EnvironmentVariableOperator for applying changes to the Environment Variables of containers
  */
 public class DeploymentConfigWatcher implements Watcher<DeploymentConfig>, Runnable {
 
-    private final static Logger logger = Logger.getLogger(DeploymentConfigWatcher.class.getName());
+    private final static org.apache.logging.log4j.Logger logger = LogManager.getLogger(DeploymentConfigWatcher.class);
```
Comment (matzew) [inline Operator/src/main/java/io/streamzi/ev/watcher/DeploymentConfigWatcher.java]: full package

File: Operator/src/main/java/io/streamzi/ev/watcher/DeploymentWatcher.java [hunk 7]
```diff
@@ -7,22 +7,21 @@
...
-import java.util.logging.Logger;
 
 /**
  * Watch for changes in a Deployment and push them to an EnvironmentVariableOperator which will look to see if any
  * ConfigMaps contain Environment Variables that should be copied into this deployment
  */
 public class DeploymentWatcher implements Watcher<Deployment>, Runnable {
 
-    private final static Logger logger = Logger.getLogger(DeploymentWatcher.class.getName());
+    private final static org.apache.logging.log4j.Logger logger = LogManager.getLogger(DeploymentWatcher.class);
```
Comment (matzew) [inline Operator/src/main/java/io/streamzi/ev/watcher/DeploymentWatcher.java]: full package

**Comment by matzew** [comment]
I'd squash all commit to just one

**Comment by sjwoodman** [comment]
Done...

**Pull request merged**